framework/src/main/java/org/checkerframework/framework/qual/CFComment.java

```diff
@@ -0,0 +1,41 @@
+package org.checkerframework.framework.qual;
+
+import static java.lang.annotation.ElementType.*;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation is to be used to add comments related to Checker-Framework. This annotation can
+ * be used on packages, classes, interfaces and any elements inside these.
+ *
+ * <p>Each comment should first start with the type-system to which comment is related to followed
+ * by ':' and then actual comment. Provide single string if comment is for only one type-system and
```

Comment:
Users should be permitted to make their own decisions about this.  In particular, it is sensible for a user to break a long comment across lines, by putting it in multiple strings.